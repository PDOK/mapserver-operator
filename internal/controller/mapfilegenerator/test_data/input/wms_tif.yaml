apiVersion: pdok.nl/v2beta1
kind: WMS
metadata:
  name: bzk-luchtfotolabels-v1-0
  labels:
    dataset-owner: bzk
    dataset: luchtfotolabels
    service-version: v1_0
    service-type: wms
spec:
  general:
    datasetOwner: bzk
    dataset: luchtfotolabels
    serviceVersion: v1_0
  kubernetes:
    autoscaling:
      minReplicas: 1
      maxReplicas: 2
    healthCheck:
      boundingbox: 135036.1077132325445,456913.9317436855054,135531.2729437439411,457377.1306112145539
    resources:
      limits:
        memory: 4G
        ephemeralStorage: 6G
      requests:
        cpu: "1"
        memory: 4G
        ephemeralStorage: 6G
  options:
    automaticCasing: false
    disableWebserviceProxy: false
    includeIngress: false
    validateRequests: false
    validateChildStyleNameEqual: false
  service:
    inspire: false
    title: Luchtfoto Labels WMS
    abstract: "De luchtfoto labels bestaan uit weglabels en wegassen en kunnen worden gebruikt als laag (overlay) op onder andere de PDOK luchtfoto."
    keywords:
      - bzk
      - luchtfotolabels
    metadataIdentifier: 70562932-e7dc-4ba2-ba4f-05863d02587c
    authority:
      name: kadaster
      url: http://www.kadaster.nl
    dataEPSG: EPSG:28992
    stylingAssets:
      configMapRefs:
        - name: ${INCLUDES}
    layers:
      - name: lufolabels
        visible: true
        title: Luchtfoto labels
        abstract: "De luchtfoto labels bestaan uit weglabels en wegassen en kunnen worden gebruikt als laag (overlay) op onder andere de PDOK luchtfoto."
        keywords:
          - bzk
          - luchtfotolabels
        datasetMetadataIdentifier: 6ca22f53-b072-42f4-b920-104c7c83cd28
        sourceMetadataIdentifier: 901647c2-802d-11e6-ae22-56b6b6499611
        styles:
          - name: luchtfotolabels
            title: Luchtfotolabels
      - name: luchtfotoroads_100pixkm
        visible: true
        title: Luchtfoto roads 100pixkm
        group: lufolabels
        abstract: "De luchtfoto labels bestaan uit weglabels en wegassen en kunnen worden gebruikt als laag (overlay) op onder andere de PDOK luchtfoto."
        keywords:
          - bzk
          - luchtfotolabels
        datasetMetadataIdentifier: 6ca22f53-b072-42f4-b920-104c7c83cd28
        sourceMetadataIdentifier: 901647c2-802d-11e6-ae22-56b6b6499611
        minScale: 24001
        maxScale: 48001
        styles:
          - name: luchtfotolabels
            title: Luchtfotolabels
            visualization: roads.style
        data:
          tif:
            blobKey: tifs/bzk/luchtfotolabels/${GPKG_VERSION}/100pixkm_luforoads/100pixkm_luforoads.vrt
            offsite: "#978E97"
            resample: BILINEAR
      - name: luchtfotoroads_200pixkm
        visible: true
        title: Luchtfoto roads 200pixkm
        group: lufolabels
        abstract: "De luchtfoto labels bestaan uit weglabels en wegassen en kunnen worden gebruikt als laag (overlay) op onder andere de PDOK luchtfoto."
        keywords:
          - bzk
          - luchtfotolabels
        datasetMetadataIdentifier: 6ca22f53-b072-42f4-b920-104c7c83cd28
        sourceMetadataIdentifier: 901647c2-802d-11e6-ae22-56b6b6499611
        minScale: 12001
        maxScale: 24001
        styles:
          - name: luchtfotolabels
            title: Luchtfotolabels
            visualization: roads.style
        data:
          tif:
            blobKey: tifs/bzk/luchtfotolabels/${GPKG_VERSION}/200pixkm_luforoads/200pixkm_luforoads.vrt
            offsite: "#978E97"
            resample: BILINEAR
      - name: luchtfotoroads_400pixkm
        visible: true
        title: Luchtfoto roads 400pixkm
        group: lufolabels
        abstract: "De luchtfoto labels bestaan uit weglabels en wegassen en kunnen worden gebruikt als laag (overlay) op onder andere de PDOK luchtfoto."
        keywords:
          - bzk
          - luchtfotolabels
        datasetMetadataIdentifier: 6ca22f53-b072-42f4-b920-104c7c83cd28
        sourceMetadataIdentifier: 901647c2-802d-11e6-ae22-56b6b6499611
        minScale: 6001
        maxScale: 12001
        styles:
          - name: luchtfotolabels
            title: Luchtfotolabels
            visualization: roads.style
        data:
          tif:
            blobKey: tifs/bzk/luchtfotolabels/${GPKG_VERSION}/400pixkm_luforoads/400pixkm_luforoads.vrt
            offsite: "#978E97"
            resample: BILINEAR
      - name: luchtfotoroads_800pixkm
        visible: true
        title: Luchtfoto roads 800pixkm
        group: lufolabels
        abstract: "De luchtfoto labels bestaan uit weglabels en wegassen en kunnen worden gebruikt als laag (overlay) op onder andere de PDOK luchtfoto."
        keywords:
          - bzk
          - luchtfotolabels
        datasetMetadataIdentifier: 6ca22f53-b072-42f4-b920-104c7c83cd28
        sourceMetadataIdentifier: 901647c2-802d-11e6-ae22-56b6b6499611
        minScale: 3001
        maxScale: 6001
        styles:
          - name: luchtfotolabels
            title: Luchtfotolabels
            visualization: roads.style
        data:
          tif:
            blobKey: tifs/bzk/luchtfotolabels/${GPKG_VERSION}/800pixkm_luforoads/800pixkm_luforoads.vrt
            offsite: "#978E97"
            resample: BILINEAR
      - name: luchtfotoroads_1600pixkm
        visible: true
        title: Luchtfoto roads 1600pixkm
        group: lufolabels
        abstract: "De luchtfoto labels bestaan uit weglabels en wegassen en kunnen worden gebruikt als laag (overlay) op onder andere de PDOK luchtfoto."
        keywords:
          - bzk
          - luchtfotolabels
        datasetMetadataIdentifier: 6ca22f53-b072-42f4-b920-104c7c83cd28
        sourceMetadataIdentifier: 901647c2-802d-11e6-ae22-56b6b6499611
        minScale: 1501
        maxScale: 3001
        styles:
          - name: luchtfotolabels
            title: Luchtfotolabels
            visualization: roads.style
        data:
          tif:
            blobKey: tifs/bzk/luchtfotolabels/${GPKG_VERSION}/1600pixkm_luforoads/1600pixkm_luforoads.vrt
            offsite: "#978E97"
            resample: BILINEAR
      - name: luchtfotolabels_100pixkm
        visible: true
        title: Luchtfoto labels 100pixkm
        group: lufolabels
        abstract: "De luchtfoto labels bestaan uit weglabels en wegassen en kunnen worden gebruikt als laag (overlay) op onder andere de PDOK luchtfoto."
        keywords:
          - bzk
          - luchtfotolabels
        datasetMetadataIdentifier: 6ca22f53-b072-42f4-b920-104c7c83cd28
        sourceMetadataIdentifier: 901647c2-802d-11e6-ae22-56b6b6499611
        minScale: 24001
        maxScale: 48001
        styles:
          - name: luchtfotolabels
            title: Luchtfotolabels
            visualization: labels.style
        data:
          tif:
            blobKey: tifs/bzk/luchtfotolabels/${GPKG_VERSION}/100pixkm_lufolabels/100pixkm_lufolabels.vrt
            offsite: "#978E97"
            resample: BILINEAR
      - name: luchtfotolabels_200pixkm
        visible: true
        title: Luchtfoto labels 200pixkm
        group: lufolabels
        abstract: "De luchtfoto labels bestaan uit weglabels en wegassen en kunnen worden gebruikt als laag (overlay) op onder andere de PDOK luchtfoto."
        keywords:
          - bzk
          - luchtfotolabels
        datasetMetadataIdentifier: 6ca22f53-b072-42f4-b920-104c7c83cd28
        sourceMetadataIdentifier: 901647c2-802d-11e6-ae22-56b6b6499611
        minScale: 12001
        maxScale: 24001
        styles:
          - name: luchtfotolabels
            title: Luchtfotolabels
            visualization: labels.style
        data:
          tif:
            blobKey: tifs/bzk/luchtfotolabels/${GPKG_VERSION}/200pixkm_lufolabels/200pixkm_lufolabels.vrt
            offsite: "#978E97"
            resample: BILINEAR
      - name: luchtfotolabels_400pixkm
        visible: true
        title: Luchtfoto labels 400pixkm
        group: lufolabels
        abstract: "De luchtfoto labels bestaan uit weglabels en wegassen en kunnen worden gebruikt als laag (overlay) op onder andere de PDOK luchtfoto."
        keywords:
          - bzk
          - luchtfotolabels
        datasetMetadataIdentifier: 6ca22f53-b072-42f4-b920-104c7c83cd28
        sourceMetadataIdentifier: 901647c2-802d-11e6-ae22-56b6b6499611
        minScale: 6001
        maxScale: 12001
        styles:
          - name: luchtfotolabels
            title: Luchtfotolabels
            visualization: labels.style
        data:
          tif:
            blobKey: tifs/bzk/luchtfotolabels/${GPKG_VERSION}/400pixkm_lufolabels/400pixkm_lufolabels.vrt
            offsite: "#978E97"
            resample: BILINEAR
      - name: luchtfotolabels_800pixkm
        visible: true
        title: Luchtfoto labels 800pixkm
        group: lufolabels
        abstract: "De luchtfoto labels bestaan uit weglabels en wegassen en kunnen worden gebruikt als laag (overlay) op onder andere de PDOK luchtfoto."
        keywords:
          - bzk
          - luchtfotolabels
        datasetMetadataIdentifier: 6ca22f53-b072-42f4-b920-104c7c83cd28
        sourceMetadataIdentifier: 901647c2-802d-11e6-ae22-56b6b6499611
        minScale: 3001
        maxScale: 6001
        styles:
          - name: luchtfotolabels
            title: Luchtfotolabels
            visualization: labels.style
        data:
          tif:
            blobKey: tifs/bzk/luchtfotolabels/${GPKG_VERSION}/800pixkm_lufolabels/800pixkm_lufolabels.vrt
            offsite: "#978E97"
            resample: BILINEAR
      - name: luchtfotolabels_1600pixkm
        visible: true
        title: Luchtfoto labels 1600pixkm
        group: lufolabels
        abstract: "De luchtfoto labels bestaan uit weglabels en wegassen en kunnen worden gebruikt als laag (overlay) op onder andere de PDOK luchtfoto."
        keywords:
          - bzk
          - luchtfotolabels
        datasetMetadataIdentifier: 6ca22f53-b072-42f4-b920-104c7c83cd28
        sourceMetadataIdentifier: 901647c2-802d-11e6-ae22-56b6b6499611
        minScale: 1501
        maxScale: 3001
        styles:
          - name: luchtfotolabels
            title: Luchtfotolabels
            visualization: labels.style
        data:
          tif:
            blobKey: tifs/bzk/luchtfotolabels/${GPKG_VERSION}/1600pixkm_lufolabels/1600pixkm_lufolabels.vrt
            offsite: "#978E97"
            resample: BILINEAR
      - name: luchtfotolabels_3200pixkm
        visible: true
        title: Luchtfoto labels 3200pixkm
        group: lufolabels
        abstract: "De luchtfoto labels bestaan uit weglabels en wegassen en kunnen worden gebruikt als laag (overlay) op onder andere de PDOK luchtfoto."
        keywords:
          - bzk
          - luchtfotolabels
        datasetMetadataIdentifier: 6ca22f53-b072-42f4-b920-104c7c83cd28
        sourceMetadataIdentifier: 901647c2-802d-11e6-ae22-56b6b6499611
        maxScale: 1501
        styles:
          - name: luchtfotolabels
            title: Luchtfotolabels
            visualization: labels.style
        data:
          tif:
            blobKey: tifs/bzk/luchtfotolabels/${GPKG_VERSION}/3200pixkm_lufolabels/3200pixkm_lufolabels.vrt
            offsite: "#978E97"
            resample: BILINEAR
