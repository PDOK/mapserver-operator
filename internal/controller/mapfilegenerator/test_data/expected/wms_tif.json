{
    "authority_url": "http://www.kadaster.nl",
    "automatic_casing": false,
    "data_epsg": "EPSG:28992",
    "dataset_owner": "kadaster",
    "epsg_list": [
        "EPSG:28992",
        "EPSG:25831",
        "EPSG:25832",
        "EPSG:3034",
        "EPSG:3035",
        "EPSG:3857",
        "EPSG:4258",
        "EPSG:4326",
        "CRS:84"
    ],
    "group_layers": [
        {
            "abstract": "De luchtfoto labels bestaan uit weglabels en wegassen en kunnen worden gebruikt als laag (overlay) op onder andere de PDOK luchtfoto.",
            "name": "lufolabels",
            "style_name": "luchtfotolabels",
            "style_title": "Luchtfotolabels",
            "title": "Luchtfoto labels"
        }
    ],
    "layers": [
        {
            "abstract": "De luchtfoto labels bestaan uit weglabels en wegassen en kunnen worden gebruikt als laag (overlay) op onder andere de PDOK luchtfoto.",
            "dataset_metadata_id": "6ca22f53-b072-42f4-b920-104c7c83cd28",
            "dataset_source_id": "901647c2-802d-11e6-ae22-56b6b6499611",
            "geometry_type": "Raster",
            "get_feature_info_includes_class": false,
            "group_name": "lufolabels",
            "keywords": "bzk,luchtfotolabels",
            "layer_extent": "-25000 250000 280000 860000",
            "maxscale": "48001",
            "minscale": "24001",
            "name": "luchtfotoroads_100pixkm",
            "offsite": "#978E97",
            "resample": "BILINEAR",
            "styles": [
                {
                    "path": "/styling/roads.style",
                    "title": "Luchtfotolabels"
                }
            ],
            "tif_path": "/srv/data/tif/100pixkm_luforoads.vrt",
            "title": "Luchtfoto roads 100pixkm"
        },
        {
            "abstract": "De luchtfoto labels bestaan uit weglabels en wegassen en kunnen worden gebruikt als laag (overlay) op onder andere de PDOK luchtfoto.",
            "dataset_metadata_id": "6ca22f53-b072-42f4-b920-104c7c83cd28",
            "dataset_source_id": "901647c2-802d-11e6-ae22-56b6b6499611",
            "geometry_type": "Raster",
            "get_feature_info_includes_class": false,
            "group_name": "lufolabels",
            "keywords": "bzk,luchtfotolabels",
            "layer_extent": "-25000 250000 280000 860000",
            "maxscale": "24001",
            "minscale": "12001",
            "name": "luchtfotoroads_200pixkm",
            "offsite": "#978E97",
            "resample": "BILINEAR",
            "styles": [
                {
                    "path": "/styling/roads.style",
                    "title": "Luchtfotolabels"
                }
            ],
            "tif_path": "/srv/data/tif/200pixkm_luforoads.vrt",
            "title": "Luchtfoto roads 200pixkm"
        },
        {
            "abstract": "De luchtfoto labels bestaan uit weglabels en wegassen en kunnen worden gebruikt als laag (overlay) op onder andere de PDOK luchtfoto.",
            "dataset_metadata_id": "6ca22f53-b072-42f4-b920-104c7c83cd28",
            "dataset_source_id": "901647c2-802d-11e6-ae22-56b6b6499611",
            "geometry_type": "Raster",
            "get_feature_info_includes_class": false,
            "group_name": "lufolabels",
            "keywords": "bzk,luchtfotolabels",
            "layer_extent": "-25000 250000 280000 860000",
            "maxscale": "12001",
            "minscale": "6001",
            "name": "luchtfotoroads_400pixkm",
            "offsite": "#978E97",
            "resample": "BILINEAR",
            "styles": [
                {
                    "path": "/styling/roads.style",
                    "title": "Luchtfotolabels"
                }
            ],
            "tif_path": "/srv/data/tif/400pixkm_luforoads.vrt",
            "title": "Luchtfoto roads 400pixkm"
        },
        {
            "abstract": "De luchtfoto labels bestaan uit weglabels en wegassen en kunnen worden gebruikt als laag (overlay) op onder andere de PDOK luchtfoto.",
            "dataset_metadata_id": "6ca22f53-b072-42f4-b920-104c7c83cd28",
            "dataset_source_id": "901647c2-802d-11e6-ae22-56b6b6499611",
            "geometry_type": "Raster",
            "get_feature_info_includes_class": false,
            "group_name": "lufolabels",
            "keywords": "bzk,luchtfotolabels",
            "layer_extent": "-25000 250000 280000 860000",
            "maxscale": "6001",
            "minscale": "3001",
            "name": "luchtfotoroads_800pixkm",
            "offsite": "#978E97",
            "resample": "BILINEAR",
            "styles": [
                {
                    "path": "/styling/roads.style",
                    "title": "Luchtfotolabels"
                }
            ],
            "tif_path": "/srv/data/tif/800pixkm_luforoads.vrt",
            "title": "Luchtfoto roads 800pixkm"
        },
        {
            "abstract": "De luchtfoto labels bestaan uit weglabels en wegassen en kunnen worden gebruikt als laag (overlay) op onder andere de PDOK luchtfoto.",
            "dataset_metadata_id": "6ca22f53-b072-42f4-b920-104c7c83cd28",
            "dataset_source_id": "901647c2-802d-11e6-ae22-56b6b6499611",
            "geometry_type": "Raster",
            "get_feature_info_includes_class": false,
            "group_name": "lufolabels",
            "keywords": "bzk,luchtfotolabels",
            "layer_extent": "-25000 250000 280000 860000",
            "maxscale": "3001",
            "minscale": "1501",
            "name": "luchtfotoroads_1600pixkm",
            "offsite": "#978E97",
            "resample": "BILINEAR",
            "styles": [
                {
                    "path": "/styling/roads.style",
                    "title": "Luchtfotolabels"
                }
            ],
            "tif_path": "/srv/data/tif/1600pixkm_luforoads.vrt",
            "title": "Luchtfoto roads 1600pixkm"
        },
        {
            "abstract": "De luchtfoto labels bestaan uit weglabels en wegassen en kunnen worden gebruikt als laag (overlay) op onder andere de PDOK luchtfoto.",
            "dataset_metadata_id": "6ca22f53-b072-42f4-b920-104c7c83cd28",
            "dataset_source_id": "901647c2-802d-11e6-ae22-56b6b6499611",
            "geometry_type": "Raster",
            "get_feature_info_includes_class": false,
            "group_name": "lufolabels",
            "keywords": "bzk,luchtfotolabels",
            "layer_extent": "-25000 250000 280000 860000",
            "maxscale": "48001",
            "minscale": "24001",
            "name": "luchtfotolabels_100pixkm",
            "offsite": "#978E97",
            "resample": "BILINEAR",
            "styles": [
                {
                    "path": "/styling/labels.style",
                    "title": "Luchtfotolabels"
                }
            ],
            "tif_path": "/srv/data/tif/100pixkm_lufolabels.vrt",
            "title": "Luchtfoto labels 100pixkm"
        },
        {
            "abstract": "De luchtfoto labels bestaan uit weglabels en wegassen en kunnen worden gebruikt als laag (overlay) op onder andere de PDOK luchtfoto.",
            "dataset_metadata_id": "6ca22f53-b072-42f4-b920-104c7c83cd28",
            "dataset_source_id": "901647c2-802d-11e6-ae22-56b6b6499611",
            "geometry_type": "Raster",
            "get_feature_info_includes_class": false,
            "group_name": "lufolabels",
            "keywords": "bzk,luchtfotolabels",
            "layer_extent": "-25000 250000 280000 860000",
            "maxscale": "24001",
            "minscale": "12001",
            "name": "luchtfotolabels_200pixkm",
            "offsite": "#978E97",
            "resample": "BILINEAR",
            "styles": [
                {
                    "path": "/styling/labels.style",
                    "title": "Luchtfotolabels"
                }
            ],
            "tif_path": "/srv/data/tif/200pixkm_lufolabels.vrt",
            "title": "Luchtfoto labels 200pixkm"
        },
        {
            "abstract": "De luchtfoto labels bestaan uit weglabels en wegassen en kunnen worden gebruikt als laag (overlay) op onder andere de PDOK luchtfoto.",
            "dataset_metadata_id": "6ca22f53-b072-42f4-b920-104c7c83cd28",
            "dataset_source_id": "901647c2-802d-11e6-ae22-56b6b6499611",
            "geometry_type": "Raster",
            "get_feature_info_includes_class": false,
            "group_name": "lufolabels",
            "keywords": "bzk,luchtfotolabels",
            "layer_extent": "-25000 250000 280000 860000",
            "maxscale": "12001",
            "minscale": "6001",
            "name": "luchtfotolabels_400pixkm",
            "offsite": "#978E97",
            "resample": "BILINEAR",
            "styles": [
                {
                    "path": "/styling/labels.style",
                    "title": "Luchtfotolabels"
                }
            ],
            "tif_path": "/srv/data/tif/400pixkm_lufolabels.vrt",
            "title": "Luchtfoto labels 400pixkm"
        },
        {
            "abstract": "De luchtfoto labels bestaan uit weglabels en wegassen en kunnen worden gebruikt als laag (overlay) op onder andere de PDOK luchtfoto.",
            "dataset_metadata_id": "6ca22f53-b072-42f4-b920-104c7c83cd28",
            "dataset_source_id": "901647c2-802d-11e6-ae22-56b6b6499611",
            "geometry_type": "Raster",
            "get_feature_info_includes_class": false,
            "group_name": "lufolabels",
            "keywords": "bzk,luchtfotolabels",
            "layer_extent": "-25000 250000 280000 860000",
            "maxscale": "6001",
            "minscale": "3001",
            "name": "luchtfotolabels_800pixkm",
            "offsite": "#978E97",
            "resample": "BILINEAR",
            "styles": [
                {
                    "path": "/styling/labels.style",
                    "title": "Luchtfotolabels"
                }
            ],
            "tif_path": "/srv/data/tif/800pixkm_lufolabels.vrt",
            "title": "Luchtfoto labels 800pixkm"
        },
        {
            "abstract": "De luchtfoto labels bestaan uit weglabels en wegassen en kunnen worden gebruikt als laag (overlay) op onder andere de PDOK luchtfoto.",
            "dataset_metadata_id": "6ca22f53-b072-42f4-b920-104c7c83cd28",
            "dataset_source_id": "901647c2-802d-11e6-ae22-56b6b6499611",
            "geometry_type": "Raster",
            "get_feature_info_includes_class": false,
            "group_name": "lufolabels",
            "keywords": "bzk,luchtfotolabels",
            "layer_extent": "-25000 250000 280000 860000",
            "maxscale": "3001",
            "minscale": "1501",
            "name": "luchtfotolabels_1600pixkm",
            "offsite": "#978E97",
            "resample": "BILINEAR",
            "styles": [
                {
                    "path": "/styling/labels.style",
                    "title": "Luchtfotolabels"
                }
            ],
            "tif_path": "/srv/data/tif/1600pixkm_lufolabels.vrt",
            "title": "Luchtfoto labels 1600pixkm"
        },
        {
            "abstract": "De luchtfoto labels bestaan uit weglabels en wegassen en kunnen worden gebruikt als laag (overlay) op onder andere de PDOK luchtfoto.",
            "dataset_metadata_id": "6ca22f53-b072-42f4-b920-104c7c83cd28",
            "dataset_source_id": "901647c2-802d-11e6-ae22-56b6b6499611",
            "geometry_type": "Raster",
            "get_feature_info_includes_class": false,
            "group_name": "lufolabels",
            "keywords": "bzk,luchtfotolabels",
            "layer_extent": "-25000 250000 280000 860000",
            "maxscale": "1501",
            "name": "luchtfotolabels_3200pixkm",
            "offsite": "#978E97",
            "resample": "BILINEAR",
            "styles": [
                {
                    "path": "/styling/labels.style",
                    "title": "Luchtfotolabels"
                }
            ],
            "tif_path": "/srv/data/tif/3200pixkm_lufolabels.vrt",
            "title": "Luchtfoto labels 3200pixkm"
        }
    ],
    "maxSize": "4000",
    "outputformat_jpg": "jpg",
    "outputformat_png8": "png",
    "service_abstract": "De luchtfoto labels bestaan uit weglabels en wegassen en kunnen worden gebruikt als laag (overlay) op onder andere de PDOK luchtfoto.",
    "service_accessconstraints": "https://creativecommons.org/publicdomain/zero/1.0/deed.nl",
    "service_extent": "-25000 250000 280000 860000",
    "service_keywords": "bzk,luchtfotolabels",
    "service_metadata_id": "",
    "service_namespace_prefix": "luchtfotolabels",
    "service_namespace_uri": "http://luchtfotolabels.geonovum.nl",
    "service_onlineresource": "https://service.pdok.nl",
    "service_path": "/bzk/luchtfotolabels/wms/v1_0",
    "service_title": "Luchtfoto Labels WMS",
    "symbols": [],
    "templates": "/srv/data/config/templates"
}